save-plugins/warn-plugin/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ kotlin {
             dependencies {
                 implementation(projects.saveCommon)
                 implementation(libs.kotlinx.serialization.core)
-                implementation("io.github.detekt.sarif4k:sarif4k")
+                implementation("io.github.detekt.sarif4k:sarif4k:0.1.0-SNAPSHOT")
```

Comment:
snapshot?

Reply:
Currently released version of `sarif4k` is `0.0.1` and the next development version in our fork is `0.1.0-SNAPSHOT`. I thought we shouldn't create release versions in GH packages (yet).